src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -1,142 +1,139 @@
-      >
-      <mat-card-subtitle
-        >The top questions we receive across all categories</mat-card-subtitle
-      >
-      <mat-card-content></mat-card-content>
-    </mat-card>
-    <hr />
-    <div *ngFor="let faqCategory of faqCategories">
+<div fxLayout="row" fxLayoutAlign="center center" class="k-header">
+  <button
+    mat-fab
+    color="primary"
+    matTooltip="Add Category"
+    (click)="onClickAddCategory()"
+  >
+    <mat-icon>add</mat-icon>
+  </button>
+  <div fxFlex class="k-header-title">FAQs</div>
+</div>
+
+<div class="mat-elevation-z8" class="p-4">
+  <div fxLayout="row">
+    <!-- Category Selector -->
+    <div fxFlex="20">
       <mat-card
         class="mb-3 clickable"
-        [ngClass]="
-          filterFaqCategory?.id == faqCategory.id ? 'background-primary' : ''
-        "
-        (click)="onClickCategory(faqCategory)"
         matRipple
+        (click)="onClickTopCategory()"
+        [ngClass]="topOnly ? 'background-primary' : ''"
       >
         <mat-card-title>
-          {{ faqCategory.title }}
-        </mat-card-title>
-        <mat-card-subtitle>
-          {{ faqCategory.description }}
-        </mat-card-subtitle>
+          <i class="fa fa-star"></i>
+          Top Questions</mat-card-title
+        >
+        <mat-card-subtitle
+          >The top questions we receive across all categories</mat-card-subtitle
+        >
         <mat-card-content></mat-card-content>
       </mat-card>
+      <hr />
+      <div *ngFor="let faqCategory of faqCategories">
+        <mat-card
+          class="mb-3 clickable"
+          [ngClass]="
+            filterFaqCategory?.id == faqCategory.id ? 'background-primary' : ''
+          "
+          (click)="onClickCategory(faqCategory)"
+          matRipple
+        >
+          <mat-card-title>
+            {{ faqCategory.title }}
+          </mat-card-title>
+          <mat-card-subtitle>
+            {{ faqCategory.description }}
+          </mat-card-subtitle>
+          <mat-card-content></mat-card-content>
+        </mat-card>
+      </div>
     </div>
-  </div>
 
-  <!-- Categories -->
-  <div class="p-2 w-60 d-flex flex-column align-items-center">
-    <!-- Title -->
-    <div class="d-flex flex-row w-100 justify-content-around position-relative">
-      <h2 class="text-center">
-        Frequently Asked Questions
-      </h2>
+    <!-- Categories -->
+    <div fxFlex="80" class="pl-3">
+      <!-- Clear Filter Button -->
       <button
-        class="absolute-right mr-2"
-        style="position: absolute;"
-        mat-mini-fab
-        (click)="onClickAddCategory()"
-        color="primary"
-        matTooltip="Add Category"
+        class="mb-3"
+        mat-raised-button
+        color="accent"
+        (click)="onClickTopCategory()"
+        *ngIf="!topOnly || filterFaqCategory"
       >
-        <mat-icon>add</mat-icon>
+        View Top Questions
       </button>
-    </div>
+      <div *ngIf="topOnly && !filterFaqCategory"></div>
```

Comment:
This was just meant as a padding/separator component. Can be removed now that we are using the fxLayout stuff.

Reply:
added to a new FAQ ticket - good catch